examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -307,6 +307,12 @@ ${ flowData.code }
 
 	}
 
+	getInstanceIndex() {
+
+		return 'gl_InstanceID';
```

Comment:
I think this would be the best way;
```suggestion
		return 'uint( gl_InstanceID )';
```

Reply:
Obvious really, I was trying to fix it at the tsl level rather than the GLSL level.
Thanks